Sources/ApodiniAudit/BestPractices/ParameterBestPractices/ReasonableParameterCount.swift

```diff
@@ -0,0 +1,70 @@
+//
+// This source file is part of the Apodini open source project
+//
+// SPDX-FileCopyrightText: 2019-2021 Paul Schmiedmayer and the Apodini project authors (see CONTRIBUTORS.md) <paul.schmiedmayer@tum.de>
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import Apodini
+
+/// BP19
```

Comment:
It would be really nice to have human readable descriptions in addition to the classification to make it easier to understand the reasoning behind the best practice design. Also applies to some other best practices 👍